spec/parallel_tests/test/runner_spec.rb

```diff
@@ -344,7 +344,7 @@ def run_with_file(content)
     it "sets process number to 2 for 1" do
       run_with_file("puts ENV['TEST_ENV_NUMBER']") do |path|
         result = call("ruby #{path}", 1, 4, {})
-        expect(result).to eq({
+        expect(result).to include({
```

Comment:
Handles additional output from the calling program about what the command was and the seed.